fix various delegate docs (#635)
harness/developer-hub: docs/platform/2_Delegates/get-started-with-delegates/delegate-requirements-and-limitations.md

![](./static/delegate-requirements-and-limitations-00

> Remember that the memory and CPU requirements are for the Delegate only. You Delegate host/pod/container will need more computing resources for its operations systems and other services such as Docker or Kubernetes.

Reviewer: ```suggestion
The requirements for memory and CPU support only the delegate. The delegate host, pod, and container require additional compute resources to support other operations, systems, and services such as Docker or Kubernetes.
```

> The Delegate runs on a Linux/UNIX container.

Reviewer: ```suggestion
The delegate runs in a Linux/UNIX container.
```

### Delegate Access Requirements

> * The Harness Delegate does NOT require root account access, but the Kubernetes and Docker Delegates run as root by default. If you do not need to install applications using INIT_SCRIPT, then you can use a non-root account or install the application without the Delegate.  

Reviewer: ```suggestion
* Delegates do not require root account access. Kubernetes and Docker delegates do, however, run as root by default. If you do not need to install applications during the initialization process (`INIT_SCRIPT`), you can use a non-root account or install the application without the delegate.  
```

See [Non-Root Delegate Installation](/docs/platform/2_Delegates/advanced-installation/non-root-delegate-installation.md).

> * If you do not run the Delegate as root, be aware that you cannot install any software using a [Delegate Initialization Script](/docs/platform/2_Delegates/delegate-reference/common-delegate-profile-scripts.md).

Reviewer: ```suggestion
* If you do not run the delegate as root, you cannot use [delegate initialization scripts](/docs/platform/Delegates/delegate-reference/common-delegate-profile-scripts.md) to install software.
```
